parkhangs-expressapp/controllers/parkController.js

```diff
@@ -24,22 +25,35 @@ const getParks = async (req, res) => {
 
 }
 
-// Adds a new event
+// Adds a new rating
 const addRating = async (req, res) => {
   var parkId = req.body.parkId;
   var rating = req.body.rating;
-    // console.log(newEvent.parkId);
 
-    try {
-        res.setHeader('Content-Type', 'application/json');
-        let inserted = await database.collection('events').update( { _id: parkId },{ $push: { "ratings": rating } });
-        assert.equal(1, inserted.insertedCount);
-        console.log('rating added');
-        return res.status(200).json({success: true, data: newEvent})
-    } catch (error) {
-        return res.status(404).json({success: false, error: 'Could not add rating'})
-      }
+  let parkToUpdate = {}
+
+  try {
+      parkToUpdate = await Park.findById(mongoose.Types.ObjectId(parkId))
+  } catch (error) {
+    return res.status(404).json({success: false,error: `Could not find the park with that id`})
+  }
+  try {
+    parkToUpdate.ratings.push(rating)
```

Comment:
are we adding the user who made the rating or no?

Reply:
makes sense, like ratingUserPair?

Reply:
something like that yeah. Maybe like "ratingObject" so at least you know it's not just the rating